src/Message/ApplePayAuthorizeRequest.php

```diff
@@ -0,0 +1,362 @@
+    }
+
+    /**
+     * Set the merchant identifier for the request
+     *
+     * @param string $url
+     * @return static
+     */
+    public function setMerchantIdentifier($value)
+    {
+        return $this->setParameter('merchantIdentifier', $value);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getDisplayName()
+    {
+        return $this->getParameter('displayName');
+    }
+
+    /**
+     * Sets the display name for the request
+     *
+     * @param string $value
+     * @return static
+     */
+    public function setDisplayName($value)
+    {
+        return $this->setParameter('displayName', $value);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getApplicationType()
+    {
+        return $this->getParameter('applicationType');
+    }
+
+    /**
+     * Sets the application type for the request
+     *
+     * @param string $value
+     * @return static
+     */
+    public function setApplicationType($value)
+    {
+        return $this->setParameter('applicationType', $value);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getApplicationUrl()
+    {
+        return $this->getParameter('applicationUrl');
+    }
+
+    /**
+     * Sets the application url for the request
+     *
+     * @param string $value
+     * @return static
+     */
+    public function setApplicationUrl($value)
+    {
+        return $this->setParameter('applicationUrl', $value);
+    }
+
+    /**
+     * Get HTTP Method.
+     *
+     * This is nearly always POST but can be over-ridden in sub classes.
+     *
+     * @return string
+     */
+    public function getHttpMethod()
+    {
+        return 'POST';
+    }
+
+    /**
+     * @return array
+     */
+    public function getData()
+    {
+        // We must have the validation URL set. If it isn't, don't make the call.
+        $this->validate('validationUrl');
+        $data = array();
+
+        // Required request parameters for Apple Session object.
+        $data['merchantIdentifier'] = $this->getMerchantIdentifier();
+        $data['displayName'] = $this->getDisplayName();
+        $data['initiative'] = $this->getApplicationType();
+        $data['initiativeContext'] = $this->getApplicationUrl();
+
+        return $data;
+    }
+
+    /**
+     * @param       $data
+     * @param array $headers
+     *
+     * @return \Guzzle\Http\Message\RequestInterface
+     */
+    protected function createClientRequest($data, array $headers = null)
+    {
+        $pemCerts = $this->getPemCertPath();
+        $keyCerts = $this->getKeyCertPath();
+        $keyCertPassword = $this->getKeyCertPassword();
+
+        // Setting options for the request.
+        $config                          = $this->httpClient->getConfig();
+        $curlOptions                     = $config->get('curl.options');
+        $curlOptions[CURLOPT_SSLVERSION] = 6;
+        $curlOptions[CURLOPT_SSLCERT] = $pemCerts;
```

Comment:
could you keep the spacing style consistent? either lining up the equals signs or just having one space is fine, but a mix is confusing